index.html

```diff
@@ -201,7 +201,7 @@ <h5 class="intro2">LET'S CONNECT</h5>
         <a href="#"><img src="images/fingers.png" alt="fingers"></a>
     </li>
  </ul>
-
+ 
```

Comment:
- [ ] please kindly follow this figma design for your desktop cards. It will make them look presentable rather than making the image full screen. please do so for all cards
figma
![Screenshot (665)](https://user-images.githubusercontent.com/65298048/206854901-06950374-16b6-43b2-919e-094d828a1f4f.png)